content/en/serverless/_index.md

```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
 
 ### 1. Install the AWS Integration
 
-Start by installing the [AWS Integration][2]. This allows Datadog to ingest Amazon CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+Install the [AWS integration][2]. This allows Datadog to ingest Lambda metrics from AWS CloudWatch. 
 
-### 2. Instrument your application
+### 2. Install the Datadog Forwarder
+
+Skip this step if you already have the Forwarder function installed as part of the [AWS integration][2] CloudFormation stack. Otherwise, install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+
+### 3. Instrument Your Application
+
+Select the Lambda runtime below for instructions to instrument your serverless application.
 
 {{< partial name="serverless/getting-started-languages.html" >}}
 
 ## Not Using AWS Lambda?
 
 ### Azure App Services
```

Comment:
```suggestion
### Azure App Service
```

Reply:
"Azure App Services" is the official product name from Azure.